File: 5-kyu/the-hashtag-generator/javascript/solution.js
function generateHashtag (str) {
    if (!(str.length == 0) && str.trim() ){
        str = str.split(/[ ,]+/).filter(Boolean);
        let word;
        for (word in str) {
            //console.log(str[word]);
            if (str[word].length >= 140) {
                return false;
            }
            else {
                let arr = str.map((word) => {
                    return word[0].toUpperCase() + word.slice(1);
                  });
                return '#'+arr.join("");
            }
        }
    }
    else {
        return false;
    }

}

File: 5-kyu/the-hashtag-generator/python/solution.py
def generate_hashtag(s):
    if not (len(s) == 0):
        s = s.split()
        for word in s:
            if len(word) > 140:
                return False
            else:
                s = [word.capitalize() for word in s]
                return "#" + "".join(s)
    else:
        return False